lib/pow/ecto/schema.ex

```diff
@@ -76,6 +76,9 @@ defmodule Pow.Ecto.Schema do
           |> new_password_changeset(attrs, @pow_config)
         end
       end
+      
+  Note that @pow_config is set by `use Pow.Ecto.Schema` which should already 
```

Comment:
I always quote module attributes (you can see in the first few paragraphs). To keep the language consistent, I would change this to:

```
Note that the macro creates the `@pow_config` module attribute.
```

Or if you wish to make it clear with the module macro:

```
Note that the `Pow.Ecto.Schema.__using__/1` macro creates the `@pow_config` module attribute.
```

Maybe it could also be `use Pow.Ecto.Schema`. I usually just refer it as “ the macro” though, without specifying that it’s the `use` macro.

Reply:
(Third revision)

It looks like what you did for `pow_user_fields/0` is what [other projects are doing](https://www.google.com/search?channel=fs&q=%22__using__%2F1%22+site%3Ahttps%3A%2F%2Fhexdocs.pm%2F&ie=utf-8&oe=utf-8).

> A macro to add fields from the @pow_fields module attribute generated in `__using__/1`

Note that one of those results is a link to the [documentation of bamboo](https://hexdocs.pm/bamboo/Bamboo.Test.html#module-note-on-sending-from-other-processes). In that link, you will find that they actually auto-link to documentation on the `__using__/1` macro.

I don't know if that method is overkill or not. I do know that a lot of new users may not know what `__using__/1` actually does; as meta-programming is often one of the last topics in Elixir documentation and books.

Reply:
Yeah, I think just having `__using__/1` may be best as it auto links and refer to the same module. There's currently [no docs on `__using__/1`](https://github.com/danschultzer/pow/blob/91923f3a7b4ecdaeaf1587159b5768a1f4e3394a/lib/pow/ecto/schema.ex#L94) but an `use` example could be added here like in the bamboo docs. Then I think it'll be pretty clear and also can educate new users :smile: